website/content/docs/bctl-reference/verify.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "verify"
+draft: false
+weight: 5
+---
+
+`bctl verify` is used to verify a blueprint file. It does require a barebone cluster to already be created via `apply` or otherwise. It can then be used to verify modifications or additions of new addons to the cluster. Verification is not entirely client-side, as when verifying certain addons cluster resources will be temporarily created.
```

Comment:
We should put the requirement of an existing cluster as a separate pre req section the way k0sctl and kubectl are done in other sections